Util/Xamarin.Build.Download/source/Xamarin.Build.Download/DownloadUtils.cs

```diff
@@ -47,6 +47,9 @@ public IEnumerable<XamarinBuildDownload> ParseDownloadItems (ITaskItem[] items)
 					//TODO we may be able to determine the kind from the server response
 					continue;
 				}
+				if (!EnsureSecureUrl (item, xbd.Url, allowUnsecureUrls)) {
+					continue;
+				}
```

Comment:
This was specifically to follow the same process of a "bad" item - we basically skip it and log an error.